angel_system/impls/detect_activities/two_stage/two_stage.py

```diff
@@ -4,6 +4,8 @@
 from torch import nn
 # from torchmetrics import MaxMetric
 # from torchmetrics.classification.accuracy import Accuracy
+import sys
+sys.path.insert(0, '/angel_workspace/angel_system/impls/detect_activities/two_stage')
```

Comment:
We should endeavor to not do this. I presume this was added as a quick fix to resolve import errors in submodules. We should instead fix those submodule imports.